### PR TITLE
cortexpy: add aarch64/arm64 builds

### PR DESCRIPTION
**Pull request by martin-g**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by martin-g** [comment]
@BiocondaBot please fetch artifacts

**Comment by martin-g** [comment]
```
 12:38:12 BIOCONDA INFO Determining expected packages for recipes/cortexpy
Adding in variants from internal_defaults
Adding in variants from /opt/mambaforge/envs/bioconda/conda_build_config.yaml
Adding in variants from /opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/bioconda_utils/bioconda_utils-conda_build_config.yaml
12:38:12 BIOCONDA INFO Nothing to be done for recipe recipes/cortexpy
12:38:12 BIOCONDA INFO BUILD SUMMARY: successfully built 0 of 1 recipes
```

For some reason nothing is being built ...
The recipe is not blacklisted.

File: recipes/cortexpy/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,9 @@ source:
 
 build:
   skip: True  # [py27 or py >= 39]
```
Comment (bgruening) [inline recipes/cortexpy/meta.yaml:13]: we have skipped building for Python 3.8 I think

Comment (martin-g) [reply recipes/cortexpy/meta.yaml:13]: @bgruening What do you mean ? Or rather, what do you suggest ?
The build fails for newer versions of Python ...

**Comment by martin-g** [comment]
@BiocondaBot please fetch artifacts

File: recipes/cortexpy/meta.yaml (hunk 1 above)
Comment (martin-g) [reply recipes/cortexpy/meta.yaml:13]: I see. So, this is the reason why nothing is built.

**Comment by martin-g** [comment]
@BiocondaBot please fetch artifacts

**Comment by martin-g** [comment]
```

mgrigorov in 🌐 euler-arm-22 in /tmp/cortexpy via 🅒 test123 took 5s 
❯ tar xvf cortexpy-0.46.5-py38hcdb9e69_6.tar.bz2                                                                                                                                                           (test123) 
info/test/test_time_dependencies.json
info/run_exports.json
info/test/run_test.sh
info/hash_input.json
info/has_prefix
info/test/run_test.py
info/index.json
info/recipe/meta.yaml.template
info/recipe/meta.yaml
info/recipe/conda_build_config.yaml
info/about.json
info/files
info/licenses/LICENSE.txt
info/paths.json
info/git
lib/python3.8/site-packages/cortexpy/graph/interactor_ext.html
lib/python3.8/site-packages/cortexpy/graph/parser/kmer_ext.html
lib/python3.8/site-packages/cortexpy/graph/parser/kmer_ext.cpython-38-aarch64-linux-gnu.so
lib/python3.8/site-packages/cortexpy-0.46.5.dist-info/direct_url.json
lib/python3.8/site-packages/cortexpy-0.46.5.dist-info/INSTALLER
lib/python3.8/site-packages/cortexpy-0.46.5.dist-info/WHEEL
lib/python3.8/site-packages/cortexpy-0.46.5.dist-info/METADATA
lib/python3.8/site-packages/cortexpy-0.46.5.dist-info/RECORD
lib/python3.8/site-packages/cortexpy-0.46.5.dist-info/REQUESTED
lib/python3.8/site-packages/cortexpy/command/exit_codes.yaml
lib/python3.8/site-packages/cortexpy/graph/parser/kmer_ext.pyx
lib/python3.8/site-packages/cortexpy/graph/parser/kmer_ext.cpp
lib/python3.8/site-packages/cortexpy-0.46.5.dist-info/top_level.txt
lib/python3.8/site-packages/cortexpy-0.46.5.dist-info/entry_points.txt
lib/python3.8/site-packages/cortexpy-0.46.5.dist-info/LICENSE.txt
lib/python3.8/site-packages/cortexpy/__init__.py
lib/python3.8/site-packages/cortexpy/test/__init__.py
lib/python3.8/site-packages/cortexpy/test/builder/__init__.py
lib/python3.8/site-packages/cortexpy/test/builder/graph/__init__.py
lib/python3.8/site-packages/cortexpy/command/__init__.py
lib/python3.8/site-packages/cortexpy/test/mock/graph.py
lib/python3.8/site-packages/cortexpy/test/constants.py
lib/python3.8/site-packages/cortexpy/test/expectation/__init__.py
lib/python3.8/site-packages/cortexpy/__main__.py
lib/python3.8/site-packages/cortexpy/constants.py
lib/python3.8/site-packages/cortexpy/test/builder/unitigs.py
lib/python3.8/site-packages/cortexpy/graph/parser/constants.py
lib/python3.8/site-packages/cortexpy/command/shared.py
lib/python3.8/site-packages/cortexpy/graph/traversal/utils.py
lib/python3.8/site-packages/cortexpy/test/expectation/links.py
lib/python3.8/site-packages/cortexpy/test/driver/graph/find_unitigs.py
lib/python3.8/site-packages/cortexpy/logging_config.py
lib/python3.8/site-packages/cortexpy/cli.py
lib/python3.8/site-packages/cortexpy/test/builder/graph/kmer.py
lib/python3.8/site-packages/cortexpy/test/builder/unitig_finder.py
lib/python3.8/site-packages/cortexpy/graph/parser/kmer_collection.py
lib/python3.8/site-packages/cortexpy/command/prune.py
lib/python3.8/site-packages/cortexpy/test/expectation/fasta.py
lib/python3.8/site-packages/cortexpy/graph/parser/random_access_collection.py
lib/python3.8/site-packages/cortexpy/utils.py
lib/python3.8/site-packages/cortexpy/graph/parser/streaming.py
lib/python3.8/site-packages/cortexpy/command/assemble.py
lib/python3.8/site-packages/cortexpy/test/driver/graph/traversal.py
lib/python3.8/site-packages/cortexpy/graph/serializer/serializer.py
lib/python3.8/site-packages/cortexpy/test/driver/graph/serializer.py
lib/python3.8/site-packages/cortexpy/graph/serializer/kmer.py
lib/python3.8/site-packages/cortexpy/test/expectation/unitig_graph.py
lib/python3.8/site-packages/cortexpy/test/expectation/kmer.py
lib/python3.8/site-packages/cortexpy/test/expectation/graph.py
lib/python3.8/site-packages/cortexpy/command/view.py
lib/python3.8/site-packages/cortexpy/test/builder/graph/graph.py
lib/python3.8/site-packages/cortexpy/test/builder/mccortex.py
lib/python3.8/site-packages/cortexpy/test/builder/graph/body.py
lib/python3.8/site-packages/cortexpy/command/subgraph.py
lib/python3.8/site-packages/cortexpy/test/expectation/json.py
lib/python3.8/site-packages/cortexpy/graph/contig_retriever.py
lib/python3.8/site-packages/cortexpy/test/runner.py
lib/python3.8/site-packages/cortexpy/test/builder/graph/header.py
lib/python3.8/site-packages/cortexpy/edge_set.py
lib/python3.8/site-packages/cortexpy/command/traverse.py
lib/python3.8/site-packages/cortexpy/test/builder/graph/cortex.py
lib/python3.8/site-packages/cortexpy/graph/traversal/branch.py
lib/python3.8/site-packages/cortexpy/graph/parser/header.py
lib/python3.8/site-packages/cortexpy/graph/parser/random_access.py
lib/python3.8/site-packages/cortexpy/links.py
lib/python3.8/site-packages/cortexpy/graph/traversal/engine.py
lib/python3.8/site-packages/cortexpy/test/driver/command.py
lib/python3.8/site-packages/cortexpy/graph/serializer/unitig.py
lib/python3.8/site-packages/cortexpy/graph/interactor.py
lib/python3.8/site-packages/cortexpy/graph/parser/kmer.py
lib/python3.8/site-packages/cortexpy/tesserae.py
lib/python3.8/site-packages/cortexpy/graph/cortex.py
lib/python3.8/site-packages/cortexpy/graph/serializer/__init__.py
lib/python3.8/site-packages/cortexpy/graph/__init__.py
lib/python3.8/site-packages/cortexpy/test/driver/graph/__init__.py
lib/python3.8/site-packages/cortexpy/graph/traversal/__init__.py
lib/python3.8/site-packages/cortexpy/graph/parser/__init__.py
lib/python3.8/site-packages/cortexpy/test/driver/__init__.py
lib/python3.8/site-packages/__init__.py
lib/python3.8/site-packages/cortexpy/test/mock/__init__.py
lib/python3.8/site-packages/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/mock/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/driver/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/parser/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/traversal/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/serializer/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/driver/graph/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/builder/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/builder/graph/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/__pycache__/constants.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/command/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/expectation/__pycache__/__init__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/mock/__pycache__/graph.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/__pycache__/__main__.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/parser/__pycache__/constants.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/command/__pycache__/shared.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/__pycache__/logging_config.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/__pycache__/constants.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/builder/__pycache__/unitigs.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/traversal/__pycache__/utils.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/__pycache__/cli.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/driver/graph/__pycache__/find_unitigs.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/command/__pycache__/prune.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/builder/graph/__pycache__/kmer.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/expectation/__pycache__/links.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/builder/__pycache__/unitig_finder.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/command/__pycache__/assemble.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/parser/__pycache__/streaming.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/__pycache__/utils.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/parser/__pycache__/kmer_collection.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/serializer/__pycache__/serializer.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/driver/graph/__pycache__/traversal.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/serializer/__pycache__/kmer.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/driver/graph/__pycache__/serializer.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/expectation/__pycache__/fasta.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/parser/__pycache__/random_access_collection.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/command/__pycache__/view.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/builder/graph/__pycache__/graph.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/__pycache__/contig_retriever.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/command/__pycache__/subgraph.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/builder/__pycache__/mccortex.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/expectation/__pycache__/kmer.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/builder/graph/__pycache__/body.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/expectation/__pycache__/graph.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/expectation/__pycache__/unitig_graph.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/command/__pycache__/traverse.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/__pycache__/runner.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/traversal/__pycache__/branch.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/builder/graph/__pycache__/header.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/__pycache__/edge_set.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/expectation/__pycache__/json.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/builder/graph/__pycache__/cortex.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/parser/__pycache__/header.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/traversal/__pycache__/engine.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/__pycache__/tesserae.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/parser/__pycache__/random_access.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/serializer/__pycache__/unitig.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/__pycache__/interactor.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/test/driver/__pycache__/command.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/__pycache__/links.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/parser/__pycache__/kmer.cpython-38.pyc
lib/python3.8/site-packages/cortexpy/graph/__pycache__/cortex.cpython-38.pyc
bin/cortexpy

mgrigorov in 🌐 euler-arm-22 in /tmp/cortexpy via 🅒 test123 
❯ file bin/* lib/python*/site-packages/*/*.so                                                                                                                                                              (test123) 

mgrigorov in 🌐 euler-arm-22 in /tmp/cortexpy via 🅒 test123 
❯ file bin/* lib/python*/site-packages/**/*.so                                                                                                                                                             (test123) 
bin/cortexpy:                                                                               a /opt/conda/conda-bld/cortexpy_1733990812062/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh script, ASCII text executable
lib/python3.8/site-packages/cortexpy/graph/parser/kmer_ext.cpython-38-aarch64-linux-gnu.so: ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, with debug_info, not stripped
```
LGTM!

**Comment by martin-g** [comment]
@BiocondaBot please add label

**Comment by martin-g** [comment]
Thank you, @mencian !

**Pull request merged**